tests/bond_active-backup/system-connections/bond0.nmconnection

```diff
@@ -17,12 +17,17 @@ use_carrier=1
 [match]
 
 [ipv4]
+dhcp-send-hostname-deprecated=false
+dhcp-send-release=0
 ignore-auto-dns=true
 method=auto
 
 [ipv6]
 addr-gen-mode=default
+dhcp-send-hostname-deprecated=false
+dhcp-send-release=0
 ignore-auto-dns=true
+ip6-privacy=1
```

Comment:
Just a note. The `<ipv6><privacy>` node is based on `net.ipv6.conf.use_tempaddr`.
In our "netconfig" tests we do not reflect `sysctl` changes till now. Also it's not that easy as they are read from:
```
              /boot/sysctl.conf-<kernelversion>
              /run/sysctl.d/*.conf
              /etc/sysctl.d/*.conf
              /usr/local/lib/sysctl.d/*.conf
              /usr/lib/sysctl.d/*.conf
              /lib/sysctl.d/*.conf
              /etc/sysctl.conf
              /etc/sysconfig/network/ifsysctl
              /etc/sysconfig/network/ifsysctl-`<interface>`
```
and especially the "common" places are often different between distros.